Source/main.py

```diff
@@ -0,0 +1,41 @@
+# Author: Leo Liu, Siena Okuno, & Isa Blancett
+# Project: Interactive Programming
+# Date: 10.30.2017
+# Description: Game setup, loop through game graphics and control, & ending call
+
+import courses
+from gameplay import Gameplay
+import sys
+import pygame
+from ending import ending
+from environment import *
+
+# INITIALIZE GAME
```

Comment:
The code below should be in a function or a class. It's always good practice to wrap things up and not have them free floating in a file.